### ISSUE TITLE
struct won't compile

### ISSUE THREAD

**Issue by ghost** [llvm-dcpu16/llvm-dcpu16#131]
Compiling the following code snippet:

```
struct person {
  int   age;
  int   height;
} adam;
```

Result in following error:

```
clang-3.1: /*/llvm-dcpu16/tools/clang/lib/CodeGen/CGRecordLayoutBuilder.cpp:481: bool
{anonymous}::CGRecordLayoutBuilder::LayoutField(const clang::FieldDecl*, uint64_t): 
Assertion `!Packed && "Could not place field even with packed struct!"' failed.
```

Might be a problem with the wide char. Not sure yet.

**Comment by ghost** [comment]
Okay, ran a quick test and it seems that the 16bit char is causing this assert. Which is odd, since the CGRecordLayoutBuilder is using the "CharUnits". That unit abstracts the character width and should prevent such things to happen.

**Comment by ghost** [comment]
I assign this issue to me, since I'm allready debugging it.

**Comment by Blei** [comment]
Should be fixed by #133.

**Comment by ghost** [comment]
Uhhh, thanks! Because I was at a loss here.

**Comment by ghost** [comment]
Fixed by commit 10790544e9d2441da92ab3593beb185ef2b08dfa
